resources/building-mobile-extensions/event-listeners.md

```diff
@@ -176,6 +186,53 @@ You can register your listener class by using the init method as in the example
 
 The following example calls the listeners hear method when a change to the Adobe Experience Platform SDKs configuration occurs.
 
+{% tabs %}
+{% tab title="Android" %}
+
+### **Android**
+
+Event listeners in Android are registered using the `registerEventListener` method of the `ExtensionApi` interface. You can access this interface by using the `getApi` method in `Extension`. This example show registering the listener from your extension contructor.
```

Comment:
Please change the second sentence so it reads as follows:

"The following example shows how to register the listener from your extension constructor."